merchants/Personas/PersonasBTCPay.md

```diff
@@ -0,0 +1,44 @@
+
+## BTCPay Server Merchant Persona
+
+While BTCPay Server as a software has plenty of [use-cases](https://docs.btcpayserver.org/UseCase/) here, our focus is on users that use it in order to accept payments. Furthermore, the main focus will be on merchants.
+
+The most logical way to distinguish merchants in BTCPay Server is to divide them in:
+- Retail Merchants (brick and mortar stores)
+- Online Merchants
+
+### Retail Merchants
+
+### Online Merchants
+
+Online merchants are persona that are using BTCPay Server to accept payments online. Their goal is to either sell a product (physical or digital), a service or accept donations.
+
+#### Bitcoin Enthusiast
+Bitcoin enthusiasts usually deploys his online store for fun and experiment. These users have technical skills and want to try BTCPay Server out. These are small shops or attempts to collect Bitcoin within a certain type of group (Bitcoin Twitter).
+
+#### Medium-size merchants
+These merchants use CMS in most of the cases for their online stores. They use BTCPay via available plugins and easily connect it to their stores.
+
+#### Privacy-orientated merchants
+
+Using BTCPay Server because of the privacy features. These merchants are usually VPN's, hosting providers, VOIP services. [Example here](https://directory.btcpayserver.org/filter/merchants/domains-hosting-vpns/)
+
+#### Enterprise
+
+An enterprise is a business that has large volume of customers. The reason they've implemented BTCPay is most likely problems they've experienced with BitPay BIP70 compatibility or simply a large portion of their customer base requested it. These enterprises commonly use a few Bitcoin payment processors either because they allow more coins or because they've used it in the past. They have developer and support departments that handle customer queries. Their integrations are custom and deeply integrated within their workflow.
+
+Example: [Cheap Air](https://www.cheapair.com/) and their [story](https://www.reddit.com/r/Bitcoin/comments/8ymzpj/update_from_cheapair_now_using_btcpay/), NameCheap their [story](https://www.namecheap.com/about/press-releases/press-releases-info/20-09-21/namecheap_now_offers_bitcoin_payment_option_via_btcpay/) [user flow](https://www.namecheap.com/support/knowledgebase/article.aspx/10312/2200/how-do-i-add-funds-to-my-namecheap-account-using-btcpay). 
+
+#### Charity
+
+Usually an organization that's trying to raise funding for a particular cause, or receive donations in Bitcoin. Larger organizations usually convert their donations to fiat by exchanging their bitcoin on an exchange. These charities have deep understanding of privacy and care about privacy. They don't pay attention to invoices, underpaid amounts or manage their invoices within BTCPay Server. They are okay with whatever amount they receive.
```

Comment:
In 2018, just as the large earthquake hit Indonesia there were a bunch of small teams popping up to assist with the relief efforts. Bitcoin was sent from overseas, and converted at IDR using a local exchange because of the speed.

These teams were not formal non-profits, just people willing to help and did not have an organisation to have wire transfers sent to.

No web pages existed until later as aftershocks and other smaller earthquakes were going on, so was mostly done through mobile wallets, social media and chat apps from friends and family overseas who were wanting to help financially.